src/browser.coffee

```diff
@@ -18,6 +17,10 @@ CoffeeScript.run = (code, options = {}) ->
   options.shiftLine = on
   Function(compile code, options)()
 
+CoffeeScript.register = ->
```

Comment:
So it seems like the strategy here is to declare this dummy method, and then in the Node version this gets overridden with the real method? This is confusing, to say the least.